### PR TITLE
Database (arshad)

### PR DESCRIPTION
**Pull request by arshadsid**
Updating readme file for some description of the repository.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,17 @@
 # FusionIIIT  
-Coming Soon
+Software Engineering project to be deployed int the institute for the automation of various functionalities, modules and tasks of the Institute that are currently being handled either manually or ineffeciently.
+A team of more than 60 people working in 12 groups with each working on a module assigned to them.
+The automated tasks includes various sections of the institute like:
+    #1) Academic database management
```
Comment (Samy-33) [inline README.md]: In markdown files you need to provide two spaces at the end of line for the next line to show up in next line.  
Life do this:  
```
+ Line 1<space><space><enter>  
+ Line 2<space><space><enter>  

Output:
+ Line 1
+ Line 2

File: README.md [hunk 2]
```diff
@@ -2,16 +2,16 @@
...
+    #4) Placement Cell
+    #5) Mess management
+    #6) Gymkhana Activities
+    #7) Scholarship and Awards Portal
+    #8) Employee Management
+    #9) Course Management
+    #10) Complaint System
+    #11) File Tracking System
+    #12) Health Center Mangement
+    #13) Visitor's Hostel Management
```
Comment (Samy-33) [inline README.md]: Rectify Spelling errors if any. And then commit again.

File: README.md (hunk 1 above)
Comment (arshadsid) [reply README.md]: ok, will change  it

File: README.md [hunk 3]
```diff
@@ -2,16 +2,16 @@
 Software Engineering project to be deployed int the institute for the automation of various functionalities, modules and tasks of the Institute that are currently being handled either manually or ineffeciently.
```
Comment (Samy-33) [inline README.md]: @arshadsid, I still see spelling error.
screenshot-2017-10-3 database arshad by arshadsid pull request 4 3peers fusioniiit

Comment (arshadsid) [reply README.md]: Changes made

**Review by Samy-33 — COMMENTED** [review]
Perfect. @arshadsid, please have a look at README of this project. You'll get better idea.  
https://github.com/Samy-33/Ozone

**Review by Samy-33 — APPROVED** [review]
Awesome. Nice PR @arshadsid

**Pull request merged**